audit-connector/src/main/java/com/sequenceiq/cloudbreak/audit/converter/AttemptAuditEventResultToGrpcAttemptAuditEventResultConverter.java

```diff
@@ -1,22 +1,47 @@
 package com.sequenceiq.cloudbreak.audit.converter;
 
-import static com.sequenceiq.cloudbreak.util.ConditionBasedEvaluatorUtil.doIfTrue;
-
-import org.apache.commons.lang3.StringUtils;
-import org.springframework.stereotype.Component;
-
 import com.cloudera.thunderhead.service.audit.AuditProto;
 import com.sequenceiq.cloudbreak.audit.model.AttemptAuditEventResult;
-import com.sequenceiq.cloudbreak.audit.model.ResultApiRequestData;
 import com.sequenceiq.cloudbreak.audit.model.ResultEventData;
-import com.sequenceiq.cloudbreak.audit.model.ResultServiceEventData;
+import com.sequenceiq.cloudbreak.util.NullUtil;
+import org.apache.commons.collections.MapUtils;
+import org.apache.commons.lang3.StringUtils;
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+import org.springframework.stereotype.Component;
+
+import java.util.LinkedList;
+import java.util.List;
+import java.util.Map;
+
+import static com.sequenceiq.cloudbreak.util.ConditionBasedEvaluatorUtil.doIfTrue;
 
 @Component
 public class AttemptAuditEventResultToGrpcAttemptAuditEventResultConverter {
 
+    private static final Logger LOGGER = LoggerFactory.getLogger(AttemptAuditEventResultToGrpcAttemptAuditEventResultConverter.class);
+
+    private final Map<Class, AuditEventDataUtilizer> auditEventDataUtilizers;
+
+    public AttemptAuditEventResultToGrpcAttemptAuditEventResultConverter(Map<Class, AuditEventDataUtilizer> auditEventDataUtilizers) {
+        if (MapUtils.isNotEmpty(auditEventDataUtilizers)) {
+            List<String> utilizers = new LinkedList<>();
+            auditEventDataUtilizers.forEach((type, utilizer) -> utilizers.add(String.format("[%s :: %s]",
+                    NullUtil.getIfNotNull(type, t -> t.getSimpleName()),
+                    NullUtil.getIfNotNull(utilizer, u -> u.getClass().getSimpleName()))));
+            String utilizerListMessage = String.join(",", utilizers);
+            LOGGER.debug("The " + AttemptAuditEventResultToGrpcAttemptAuditEventResultConverter.class.getSimpleName() + " has got the following "
+                    + AuditEventDataUtilizer.class.getSimpleName() + " implementations: {}", utilizerListMessage);
+        } else {
+            LOGGER.debug("The " + AttemptAuditEventResultToGrpcAttemptAuditEventResultConverter.class.getSimpleName() + " has not got any "
+                    + AuditEventDataUtilizer.class.getSimpleName() + " implementation!");
+        }
```

Comment:
Is this logging needed? If you insist on this logging, please move it to the Bean definition method in the Configuration class.